scripts/build-charts-ci.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+
+# Install chartmuseum using the gofish package manager
+curl -fsSL https://raw.githubusercontent.com/fishworks/gofish/main/scripts/install.sh | bash
+gofish init
+gofish install chartmuseum
```

Comment:
Can't we do this instead? https://github.com/helm/chartmuseum#helm-chart

This script will have to assume there is a Kubernetes cluster available, but we always have a cluster when running CI jobs right? We just have to make sure we run this script when that cluster is up and running.

Reply:
I already tried to install chartmuseum with its chart in k3d cluster yesterday and it didn't work very well. I didn't want to spend more time on it as it's an ephemeral stack. I can try again but it will take time.

Reply:
fair